qiskit_nature/logging.py

```diff
@@ -0,0 +1,140 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Qiskit Nature Logging Utilities."""
+
+from typing import Optional, Dict, List
+import os
+import sys
+import logging
+
+
+class DefaultStreamHandler(logging.StreamHandler):
+    """Default stream Handler"""
+
+    def __init__(self):
+        """
+        Initialize the handler.
+        """
+        super().__init__(stream=sys.stdout)
+        self.setFormatter(logging.Formatter(fmt="%(asctime)s:%(name)s:%(levelname)s: %(message)s"))
+
+
+class QiskitNatureLogging:
+    """Qiskit Nature Logging."""
```

Comment:
Would it be useful here in the docstring to say a bit more around the intent of the class and perhaps have some common example usage for how one might typically set things to show users how to make use of this class?